src/sdk.ts

```diff
@@ -99,7 +109,8 @@ export class DefaultRelayingServices implements RelayingServices {
         this.relayProvider = provider;
         this.contracts = new Contracts(
             this.web3Instance,
-            await this.web3Instance.eth.getChainId()
+            await this.web3Instance.eth.getChainId(),
+            contractAddresses
```

Comment:
I think we should check if `contractAddresses` is defined (being optional):
```
contractAddresses ?? getContractAddresses(chainId ?? DEFAULT_NETWORK_ID);
```

Reply:
This code was added by @JONAF2103 ,  i have no knowledge of this.

Reply:
@wilsoniovlabs

we can still implement @antomor's suggestion, no?

Reply:
It is fixed with merge the parameter contractAddresses with the get contracts.